apps/gitness/src/components-v2/app-shell.tsx

```diff
@@ -223,12 +225,15 @@ export const AppShell = () => {
 
 export const AppShellMFE = memo(() => {
   useRepoImportEvents()
+  const { breadcrumbs } = useGetBreadcrumbs()
 
   return (
     <>
-      <MainContentLayout breadcrumbs={<BreadcrumbsMFE />} className="min-h-screen text-cn-foreground-2">
-        <Outlet />
-      </MainContentLayout>
+      <div className="text-cn-foreground-2 flex flex-col">
```

Comment:
I wonder if you even need `flex flex-col` since stacking is the standard behaviour anyway for block elements 🤔

Reply:
do we need this extra div wrapper? can't we just pass the same classes and apply them to `MainContentLayout` directly?

Reply:
I double-checked, and I was able move the `className` to the `MainContentLayout` and get rid of an extra `div`